ranger/config/rc.conf

```diff
@@ -29,6 +29,9 @@ set viewmode miller
 # How many columns are there, and what are their relative widths?
 set column_ratios 1,3,4
 
+# Hide infostring in the parent directories column?
+set hide_infostring_in_parent_directories false
```

Comment:
I don't like the double negative. Let's flip this to show and default to true.

Reply:
Just flipped the hide to show.